server/controllers/appointmentsController.ts

```diff
@@ -119,7 +119,7 @@ export const handleAppointmentNotify = (popService = getPopService()): RequestHa
     try {
       const { cancellationReason } = req.body
       if (!cancellationReason) {
-        req.session.errorMessage = 'You must select a reason for cancelling'
+        req.session.errorMessage = 'Please select a reason for not attending'
```

Comment:
According to Carol the systems never say please! 
Can you adjust working to 'You must select a reason for not attending'